Makefile.release

```diff
@@ -0,0 +1,88 @@
+SHELL := /bin/bash
+
+COMPOSER_BIN := $(shell command -v composer 2> /dev/null)
+ifndef COMPOSER_BIN
+    $(error composer is not available on your system, please install composer)
+endif
+YARN := $(shell command -v yarn 2> /dev/null)
```

Comment:
I didn't see yarn being used in the makefile - can you quickly share why you want to keep the check in the makefile?

Reply:
Ok, we do not need that.

Reply:
done